test/prop_hex_tarball.erl

```diff
@@ -0,0 +1,16 @@
+-module(prop_hex_tarball).
+-include_lib("proper/include/proper.hrl").
+
+prop_symmetric() ->
+    ?FORALL(Binary, binary(),
```

Comment:
you should probably `collect(byte_size(Binary), ...)` here and see what you get. I think chances are you'll mostly get short binaries and the test could be more trustworthy by toying with the generator size to get something that may reach a few MBs here and there.

Reply:
good catch. I got at most 40 bytes on a recent run. Will investigate and improve later.

thanks for this and your other suggestions!

Reply:
you can try doing something like `?SIZED(Size, resize(Size*100, binary()))` to get 100x longer values based on the current generator size, so you can still get small binaries, but larger ones as well. Play with such values to get broader generation.